content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -81,134 +81,83 @@ you can view processes in other containers.
 ### Examples
 
 {{< note >}}
-The examples in this section require the `EphemeralContainers` [feature
-gate](/docs/reference/command-line-tools-reference/feature-gates/) to be
-enabled, and Kubernetes client and server version v1.16 or later.
+The examples in this section require Kubernetes server components of v1.16
+or later with the `EphemeralContainers` [feature gate](
+/docs/reference/command-line-tools-reference/feature-gates/) enabled and
+`kubectl` version v1.18 or later.
 {{< /note >}}
 
-The examples in this section demonstrate how ephemeral containers appear in
-the API. You would normally use a `kubectl` plugin for troubleshooting that
-automates these steps.
-
-Ephemeral containers are created using the `ephemeralcontainers` subresource
-of Pod, which can be demonstrated using `kubectl --raw`. First describe
-the ephemeral container to add as an `EphemeralContainers` list:
-
-```json
-{
-    "apiVersion": "v1",
-    "kind": "EphemeralContainers",
-    "metadata": {
-            "name": "example-pod"
-    },
-    "ephemeralContainers": [{
-        "command": [
-            "sh"
-        ],
-        "image": "busybox",
-        "imagePullPolicy": "IfNotPresent",
-        "name": "debugger",
-        "stdin": true,
-        "tty": true,
-        "terminationMessagePolicy": "File"
-    }]
-}
+The `kubect alpha debug` command supports adding ephemeral containers to a
+running Pod. The `pause` container image makes a good example because it is
+built from `SCRATCH`.
+
+```shell
+kubectl run pause --image=k8s.gcr.io/pause:3.1 --restart=Never
```

Comment:
Even if the example uses `k8s.gcr.io/pause`, I recommend calling the Pod something like `ephemeral-demo` and not `pause`. New readers might get confused about what `pause` means or why it's used.